flathub.json

```diff
@@ -0,0 +1,146 @@
+{
+    "app-id": "io.github.louis77.tuner",
```

Comment:
why does this need a rename the original one is verified https://flathub.org/apps/com.github.louis77.tuner

Reply:
I didn't even see the orignal name mentionned here.